_episodes/11-distribution.md

```diff
@@ -0,0 +1,270 @@
+---
+title: "Distributing Packages"
+teaching: 30
+exercises: 60
+questions:
+- "How can you get your project out there?"
+objectives:
+- "Discuss community strategies"
+keypoints:
+- ""
```

Comment:
```suggestion
- "Add package metadata and build a distribution to share a package through community platforms."
- "Adopt common conventions and establish clear expectations for compatibility versus breaking changes so that prospective users of your package have confidence in its stability and maturity."
- "Multiple packaging systems exist, and require separate preparation."
```